org-code-javabuilder/playground/src/main/java/org/code/playground/PlaygroundMessageHandler.java

```diff
@@ -35,11 +37,18 @@ public void sendMessage(PlaygroundMessage message) {
     }
   }
 
-  protected void enableMessages() {
+  @Override
+  public void exit() {
+    // TODO: Once batching is added, send any remaining messages here
+  }
+
+  @Override
+  public void enableMessages() {
```

Comment:
These need to be public now? Is it because the interface is public (if so, why does the interface need to be public?) Bear with me, just trying to get my Java know-how up :) Maybe general question, how much do you think about access modifiers when writing Java code? Are you not concerned about it sometimes (and questions like this one are too nitty in those cases)?